### ISSUE TITLE
Site build is broken

### ISSUE THREAD

**Issue by Daniel15** [yarnpkg/website#626]
@Haroenv, do you know anything about this?
```
11:47:11 AM: ./scripts/normalize-translations.rb:33:in `read': Is a directory @ io_fread - ./lang/../node_modules/fs.realpath (Errno::EISDIR)
from ./scripts/normalize-translations.rb:33:in `block (2 levels) in <main>'
from ./scripts/normalize-translations.rb:29:in `each'
from ./scripts/normalize-translations.rb:29:in `block in <main>'
from ./scripts/normalize-translations.rb:26:in `each'
from ./scripts/normalize-translations.rb:26:in `<main>'
11:47:11 AM: make[1]: *** [crowdin-download] Error 1
make: *** [build-production] Error 2
```

https://app.netlify.com/sites/yarnpkg/deploys/59aeef967960b173b1354fe6

**Comment by Haroenv** [comment]
Not exactly sure what the cause of that was, saw it a few times. It works after retrying usually, but it's still really annoying. It's got to do with pr 541 still I think

**Comment by Haroenv** [comment]
c23f92e2f03446a48be214d817d31296a235d2df broke, but fe7611b9e1d5e67401772521174d9ede3e2a80ab deployed. I'll leave this open because the translation script seems to fail almost 50% of the time and I don't really know why.

**Comment by Haroenv** [comment]
fixed in #636
